microsoft-edge/devtools-guide-chromium/performance-monitor/performance-monitor-tool.md

```diff
@@ -51,4 +51,4 @@ The **Performance monitor** tool shows three performance metrics by default, and
 
 To enable or disable any of the available performance metrics, click the labels in the side bar.
```

Comment:
```suggestion
To enable or disable any of the available performance metrics, click the labels in the side bar:
```
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/performance-monitor/performance-monitor-tool?branch=pr-en-us-2938#select-performance-metrics-to-monitor

Reply:
done